README.md

```diff
@@ -70,12 +70,12 @@ This action sets up a Julia environment for use in actions by downloading a spec
     # never: Never print versioninfo
     #
     # Default: false
-    show-versioninfo: ''
+    show-versioninfo: 'false'
 
     # Set the path to the project directory or file to use when resolving some versions (e.g. `min`).
     #
     # Defaults to using JULIA_PROJECT if defined, otherwise '.'
-    project: ''
+    project: '.'
```

Comment:
Should be JULIA_PROJECT?

Reply:
Yeah, this one is tricky, because it's hard to sum up the behavior pithily.

Using GitHub Actions syntax, I suppose it would look like this:

```yaml
project: ${{ env.JULIA_PROJECT == 'refs/heads/main' && '.' || env.JULIA_PROJECT }}
```

But I don't know if that's how we actually implement it. It seems like it'd be misleading to give an implementation in the README.

Also, the above isn't actually correct, because it doesn't distinguish between JULIA_PROJECT being unset vs being set to an empty string. So I don't think we should do that.

Reply:
Since they are still commits, you can be semi-free form
```suggestion
    project: ${{ env.JULIA_PROJECT }} or '.' (if JULIA_PROJECT is unset)
```